core/controllers/analytics.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
Actually let's just put this controller at the bottom of base.py, since it's a standard thing for all pages. I don't think we need a whole new file just for it...

Reply:
Okay